docs/usage-guide/topics/ch06-security-policies.md

```diff
@@ -42,7 +42,10 @@ The following chart maps the security policy version to protocol version and cip
 
 The "default", "default_tls13", and "default_fips" versions are special in that they will be updated with future s2n-tls changes to keep up-to-date with current security best practices. Ciphersuites, protocol versions, and other options may be added or removed, or their internal order of preference might change. **Warning**: this means that the default policies may change as a result of library updates, which could break peers that rely on legacy options.
 
-In contrast, numbered or dated versions are fixed and will never change.
+In contrast, numbered or dated versions are fixed and will never change. The numbered equivalents of the default policies are currently:
+* "default": "20170210"
+* "default_tls13": "20240417"
```

Comment:
Is it necessary to add the `default_tls13` numbered equivalent, since people have to explicitly opt in to it?

Honestly, given the troubles that default policies have caused us I'd be in favor of nuking all mentions of the `default_tls13` policy.

Reply:
How you enable it isn't really the problem. The problem is that it can change. So if we want to make changes to default_tls13 like removing 1.0 and 1.1, and we tell customers "default_tls13 can change, use a numbered policy if that's not okay for your use case", then we need a numbered equivalent of "default_tls13" for them to switch to.

We'd need that even if we erased mentions of default_tls13 in the documentation.

Reply:
> and we tell customers "default_tls13 can change, use a numbered policy if that's not okay for your use case"

My worry is more along the lines of "no one should be using default_tls13, and creating a numbered policy just encourages them to stay on a bad policy". I'm more willing to do this for people who just happened to end up on their policies (`default`, `default_fips`), but ideally people who already know how to change their policy could pick a better one.

Reply:
It's hard to define "better" though. Some customers have legitimate use cases for needing the options in "default_tls13", including 1.0 and 1.1. For a definition of "better" that assumes you customers are generic and modern, our documentation currently recommends 20230317.